concrete/controllers/single_page/dashboard/system/multilingual/setup.php

```diff
@@ -157,6 +158,10 @@ public function set_default()
                 }
                 $siteConfig->save('multilingual.default_source_locale', $defaultSourceLocale);
                 $siteConfig->save('multilingual.always_track_user_locale', $this->post('alwaysTrackUserLocale') ? true : false);
+                #new code
```

Comment:
We don't need these `#new code#` and `#end new code#` comments (otherwise, over the years we would have comments like this for every block of code 😉)